trainingController.js

```diff
@@ -0,0 +1,37 @@
+const Training = require('../model/Training')
+
+const handleNewTraining = async (req, res) => {
+    const { userName, email, description, trainingType, price, startTime, endTime } = req.body;
+    if (!userName || !email || !description || !trainingType || !price, !startTime, !endTime) {
```

Comment:
!price, !startTime, !endTime => what does this do?

Reply:
Move this check into a Validator module

Reply:
> !price, !startTime, !endTime => what does this do?

Checks if they are empty first before adding to the database